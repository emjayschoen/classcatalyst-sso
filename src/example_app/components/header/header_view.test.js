import React from 'react';
import { shallow } from 'enzyme';
import HeaderView from './header_view';

describe('<HeaderView />', () => {
  let subject;

  describe('signed in', () => {
    beforeEach(() => {
      subject = shallow(<HeaderView signedIn />);
    });

    it('renders correctly', () => {
      expect(subject).toMatchSnapshot();
    });
  });
  describe('signed out', () => {
    beforeEach(() => {
      subject = shallow(<HeaderView signedIn={false} />);
    });

    it('renders correctly', () => {
      expect(subject).toMatchSnapshot();
    });
  });
});
