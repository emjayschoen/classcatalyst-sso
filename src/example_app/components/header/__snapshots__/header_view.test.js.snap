// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderView /> signed in renders correctly 1`] = `
<styled.div>
  <header>
    <h2>
      Example App
    </h2>
    <Link
      to="/signout"
    >
      Sign Out
    </Link>
  </header>
</styled.div>
`;

exports[`<HeaderView /> signed out renders correctly 1`] = `
<styled.div>
  <header>
    <h2>
      Example App
    </h2>
    <Link
      to="/signin"
    >
      Sign In
    </Link>
  </header>
</styled.div>
`;
